server/catgenome/src/main/resources/database/catgenome/h2/v2021.08.18_20.00__homolog_database.sql

```diff
@@ -0,0 +1,7 @@
+CREATE SEQUENCE IF NOT EXISTS CATGENOME.S_HOMOLOG_DATABASE START WITH 1 INCREMENT BY 1;
```

Comment:
Could you please join all migrations into one SQL file and move it to current date?